translations/player_turns.tex/pl.po

```diff
@@ -118,6 +148,12 @@ msgid ""
 "When you move through a Field with an allied Hero, do not \\hyperlink{Categories}{Visit} the Field that the allied Hero is standing on.\\par\n"
```

Comment:
nie wiem czy nie warto byłoby tego poprawić w wersji en? z `do not` na `cannot`

Reply:
Dobrze jest, tryb rozkazujący.